src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts

```diff
@@ -291,9 +291,8 @@ describe('Sectioned form component', () => {
 
     // check section is required
     tabs = el.querySelectorAll('sky-vertical-tab');
```

Comment:
This variable is no longer needed, right?

Reply:
The first use of it is still needed -but you are right that it doesn't need set here again. I've removed that.